tests/test_json_compdb.py

```diff
@@ -49,6 +48,7 @@
     }
 ]
 nonexistent_files = ['compile_commands.json', 'nonexistent.json']
+stdout_filenames = ['-', '']
```

Comment:
Added new tests for stdout and refactored existing tests:
- Renamed `assert_compdb_equals` function to `assert_compdb_file_equals`
- Extracted compilation database compare to separate `assert_compdb_equals` function